packages/wrangler/src/config/environment.ts

```diff
@@ -1211,6 +1222,13 @@ export type TailConsumer = {
 	environment?: string;
 };
 
+export type StreamingTailConsumer = {
+	/** The name of the service streaming tail events will be forwarded to. */
+	service: string;
+	/** (Optional) The environment of the service. */
+	environment?: string;
```

Comment:
We shouldn't be adding this for new things. This `environment` field refers to service environments, which are long deprecated